Telepathy/Common.cs

```diff
@@ -83,8 +83,6 @@ protected static bool SendMessage(NetworkStream stream, byte[] content)
                 Array.Copy(content, 0, payload, header.Length, content.Length);
                 stream.Write(payload, 0, payload.Length);
 
-                // flush to make sure it is being sent immediately
-                stream.Flush();
```

Comment:
are you sure we don't need this even if nodelay=true?

Reply:
also, when leaving it in, won't this simply not matter with nagle, but still work as intended without nagle?

Reply:
I will test,  but flush sends the data immediately,  thus prevents nagle algorithm.

Reply:
ok, please let me know when tested.
perhaps we need a 'if nodelay then stream.flush' there